zebra/zebra_evpn_mh.c

```diff
@@ -2155,26 +2156,21 @@ void zebra_evpn_if_es_print(struct vty *vty, struct zebra_if *zif)
 	bool vty_print = false;
 
 	mh_buf[0] = '\0';
-	snprintf(mh_buf + strlen(mh_buf), sizeof(mh_buf) - strlen(mh_buf),
-		 "  EVPN-MH:");
+	strlcat(mh_buf, "  EVPN-MH:", sizeof(mh_buf));
 	if (zif->es_info.lid || !is_zero_mac(&zif->es_info.sysmac)) {
 		vty_print = true;
 		snprintf(
-			mh_buf + strlen(mh_buf),
-			sizeof(mh_buf) - strlen(mh_buf),
+			mh_buf, sizeof(mh_buf) - strlen(mh_buf),
```

Comment:
won't this blow away the "EVPN-MH:" above?

Reply:
it does; i don't know how I ended up dropping "+ strlen(mh_buf)". Will fix that.